dbms/src/Debug/MockRaftStoreProxy.cpp

```diff
@@ -381,6 +381,37 @@ std::tuple<uint64_t, uint64_t> MockRaftStoreProxy::normalWrite(
     return std::make_tuple(index, term);
 }
 
+std::tuple<uint64_t, uint64_t> MockRaftStoreProxy::rawWrite(
+    UInt64 region_id,
+    std::vector<std::string> && keys,
+    std::vector<std::string> && vals,
+    std::vector<WriteCmdType> && cmd_types,
+    std::vector<ColumnFamilyType> && cmd_cf)
+{
+    uint64_t index = 0;
+    uint64_t term = 0;
+    {
+        auto region = getRegion(region_id);
+        assert(region != nullptr);
+        // We have a new entry.
+        index = region->getLatestCommitIndex() + 1;
+        term = region->getLatestCommitTerm();
+        // The new entry is committed on Proxy's side.
+        region->updateCommitIndex(index);
+        // We record them, as persisted raft log, for potential recovery.
+        region->commands[index] = {
+            term,
+            MockProxyRegion::RawWrite{
+                keys,
+                vals,
+                cmd_types,
+                cmd_cf,
+            }};
```

Comment:
I think `RawWrite` is more general than `NormalWrite`, maybe we can move the line `auto key = RecordKVFormat::genKey(table_id, keys[i], 1);` into `MockRaftStoreProxy::normalWrite` and get these two branches  merged into one 
https://github.com/pingcap/tiflash/blob/5b3a8b9a0346c030e47b9309345604f7cd094e2f/dbms/src/Debug/MockRaftStoreProxy.cpp#L461-L518

Reply:
fixed